api/ownership.go

```diff
@@ -190,12 +190,14 @@ func (o *Ownership) Update(newownerInfo *Ownership, user *auth.UserInfo) error {
 	} else {
 		// Auth is enabled
 
-		// Only the owner or admin can change the group
-		if user.Username != o.Owner && !o.IsAdminByUser(user) {
+		// Only the owner, user with access type admin,
+		// or admin can change the group
+		if user.Username != o.Owner &&
+			!o.IsAdminByUser(user) &&
+			!o.IsPermitted(user, Ownership_Admin) {
 			return status.Error(codes.PermissionDenied,
-				"Only owner can update volume acls")
+				"Only owner or those with admin access type update volume acls")
```

Comment:
nit: maybe add `can` before update to make this error a little more clear

Only owner or those with admin access type **can** update volume acls

Reply:
sounds good!